arches/app/models/resource.py

```diff
@@ -428,7 +428,8 @@ def get_documents_to_index(self, fetchTiles=True, datatype_factory=None, node_da
         document["numbers"] = []
         document["date_ranges"] = []
         document["ids"] = []
-        document["provisional_resource"] = "true" if sum([len(t.data) for t in tiles]) == 0 else "false"
+        tiles_have_authoritative_data = any([any(val for val in t.data.values()) for t in tiles])
```

Comment:
Adding authoritative tiles with values testing false like `False` or `[]` won't meet this check.
```suggestion
        tiles_have_authoritative_data = any(any(val is not None for val in t.data.values()) for t in tiles)
```

Reply:
That's a good consideration, but I think we want nodes with False to not meet this check because they can be legit boolean values and empty lists should get converted to `None` with the datatype's 'clean' method.

Reply:
I stored a boolean value of False (authoritative tile), and then set a breakpoint at this line, and the value of `tiles_have_authoritative_data` was False. My thought was that that would be unexpected?

Reply:
Ah, sorry. I should have more carefully looked at your suggested change. That's right, we should be checking against `None`